### PR TITLE
Add simple unit test for transactions

### PR DESCRIPTION
**Pull request by tomaszdurka**

### CODE REVIEW

**Comment by tomaszdurka** [comment]
@fvovan please review

**Comment by fvovan** [comment]
@vogdb reworked transactions here https://github.com/cargomedia/cm-janus/pull/52
probably you need to merge these changes or discuss it in more detail with him

**Comment by tomaszdurka** [comment]
Please reivew other parts.

**Comment by tomaszdurka** [comment]
What is the state of the other pr?

File: lib/transactions.js [hunk 1]
```diff
@@ -32,22 +45,29 @@ Transactions.prototype.execute = function(id, message) {
...
     default:
-      var transaction = this.find(id);
-      if (transaction) {
-        return transaction.call(null, message).finally(function() {
-          this.remove(id);
-        }.bind(this));
-      } else {
-        return Promise.reject(new Error('unknown transaction'));
-      }
+      return this.find(id).call(null, message).finally(function() {
```
Comment (fvovan) [inline lib/transactions.js]: can lead to NPE here as `find` can return nulls

Comment (tomaszdurka) [inline lib/transactions.js]: It will be checked before. I think it's fine if it throws?

Comment (tomaszdurka) [inline lib/transactions.js]: I can throw 'Transaction not found' though.

**Comment by fvovan** [comment]
lgtm

**Pull request merged**